### PR TITLE
feat: support idempotency keys

### PR DESCRIPTION
**Pull request by sishbi**
## **Associated JIRA tasks**

BAM-56

## **What the code does.**

feat: support idempotency keys 
- PayRequest: orderId - sent by client to prevent duplicate payments, same value returned by server 
- RefundRequest: idempotencyKey - sent by client to prevent duplicate refunds

## **Does this code change break backwards compatibility?.**

.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,6 +185,7 @@ Request parameters:
 - `showTips` - (optional) whether to show (true) or hide (false) the tip options. Default is (false)
 - `payment_method` - (optional) Settings to enable going straight to QR scanning
   - `payment_method_type` - Payment method type: CARD (default), ALIPAY, WECHAT
```
Comment (endrec) [inline README.md:187]: Unrelated, but are these two really snake_case? All the rest are CamelCase. 🤔

Comment (sishbi) [reply README.md]: Good point! They should all be camel case... good catch!

Comment (sishbi) [reply README.md]: Fixed...

**Review by endrec — APPROVED** [review]
LGTM

**Pull request merged**